IIS-Administration/docfx.json

```diff
@@ -30,7 +30,20 @@
     "overwrite": [],
     "externalReference": [],
     "globalMetadata": {},
-    "fileMetadata": {},
+    "globalMetadata": {
```

Comment:
you also need author and ms.author values and I'm seeing that all content is missing required metadata like title, description and ms.date.